extensions/2.0/Khronos/KHR_XMP/README.md

```diff
@@ -0,0 +1,141 @@
+| ResourceRef       | Object, we refer to the XMP documentation for a description of its properties |
+| URL               | String formatted as described in the XMP documentation |
+| Rational          | String formatted as described in the XMP documentation |
+| FrameRate         | String formatted as described in the XMP documentation |
+| FrameCount        | String formatted as described in the XMP documentation |
+| Part              | String formatted as described in the XMP documentation |
+
+### Additional Properties
+
+| XMP Property Type | JSON Type |
+| ----------------- | --------- |
+| Array             | Array     |
+| [ResourceEvent](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/ResourceEvent.md)| Object, we refer to the XMP documentation for a description of its properties |
+
+
+## The XMP gltf Namespace
+The `gltf` namespace is used to extend `XMP` to include metadata of interest to the glTF working group.
+The table below describes the properties in the `gltf` namespace.
+
+| Property Name  | Desrcription | JSON Type    |
+| -------------- | ------------ | ------------ |
+| spdx           | A [SPDX](https://spdx.org/licenses/) license identifier   | string |
+
+
+## Defining XMP Metadata
+An indirection level is introduced to avoid replicating the same XMP metadata in multiple glTF nodes.
+XMP metadatas are defined within a dictionary property in the glTF scene description file, by adding an extensions property to the top-level glTF 2.0 object and defining a KHR_XMP property with a `xmps` array inside it.
+The following example defines a glTF scene with a sample XMP metadata.
+
+```
+"extensions": {
+    "KHR_XMP" : {
+        "xmps" : [
+            {
+                "dc:contributor" : [ "Creator1Name", "Creator2Email@email.com", "Creator3Name<Email@email.com>"],
+                "dc:coverage" : "Bay Area, California, United States",
+                "dc:creator" : [ "CreatorName", "CreatorEmail@email.com"],
+                "dc:date" : ["1997-07-16T19:20:30+01:00"],
+                "dc:description" : { "en-us": "text"},
+                "dc:format" : "model/gltf-binary",
+                "dc:identifier" : "urn:stock-id:292930",
+                "dc:language" : ["en"],
+                "dc:publisher" : ["Company"],
+                "dc:relation" : ["https://www.khronos.org/"],
+                "dc:rights" : {"en-us": "BSD"},
+                "dc:source" : "http://related_resource.org/derived_from_this.gltf",
+                "dc:subject" : ["architecture"],
+                "dc:title" : { "en-us" : "MyModel", "it-IT" : "Mio Modello"},
+                "dc:type" : ["Physical Object"],
+                "gltf:spdx" : "BSD-Protection",
+            }
+        ]
+    }
+}
+```
+
+
+## Instantiating XMP metadata
+Metadata can be instantiate in a node by defining the extensions.KHR_XMP property and, within that, an index into the `xmps` array using the `xmp` property.
```

Comment:
From the XMP spec:

> An  instance  of  the  XMP  data  model  is  called  an  XMP  packet. 

Would it be correct usage to use `packet` and `packets` here? This seems clearer and more aesthetic than xmp(s).

Reply:
I agree, thank you for the suggestion.